cypress/integration/examples/network_requests.spec.js

```diff
@@ -0,0 +1,201 @@
+/// <reference types="cypress" />
```

Comment:
The given task requires splitting the test flow into three tests and creating a custom command to log in using the API. This is currently not implemented.